rule-packs/aws-config.json

```diff
@@ -921,7 +921,7 @@
   },
   {
     "name": "ec2-security-group-attached-to-eni",
-    "description": "Checks that non-default security groups are attached to EC2 instances or an ENI.",
+    "description": "Checks that any non-default security groups are in use and attached to EC2 instances or an ENI.",
```

Comment:
The J1QL includes '!PROTECTS' which indicates that the non-default SecGroups returned are not configured to protect any EC2 instances or ENIs.. they could be configured to protect another resource type (or none at all)

A more accurate description could be "Checks for any existing non-default security groups which are not configured to protect an EC2 instance or an ENI."